<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OutputFileForm</class>
 <widget class="QWidget" name="OutputFileForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>824</width>
    <height>202</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="add_param_btn">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>114</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Add Arg</string>
   </property>
  </widget>
  <widget class="QFETableWidget" name="outputparam_table">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>10</y>
     <width>681</width>
     <height>141</height>
    </rect>
   </property>
   <property name="dragDropMode">
    <enum>QAbstractItemView::DragDrop</enum>
   </property>
   <property name="defaultDropAction">
    <enum>Qt::MoveAction</enum>
   </property>
   <property name="selectionBehavior">
    <enum>QAbstractItemView::SelectRows</enum>
   </property>
   <attribute name="horizontalHeaderStretchLastSection">
    <bool>true</bool>
   </attribute>
   <column>
    <property name="text">
     <string>Parameter</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Value</string>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="delete_param_btn">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>114</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Delete Arg</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="outputfile_le">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>160</y>
     <width>521</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>170</y>
     <width>161</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Output File Name</string>
   </property>
  </widget>
  <widget class="QPushButton" name="set_dir_btn">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>114</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Output Dir</string>
   </property>
  </widget>
  <widget class="QPushButton" name="outputfile_btn">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>160</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QFETableWidget</class>
   <extends>QTableWidget</extends>
   <header>qfetablewidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
